packages/lodestar/src/network/globals.ts

```diff
@@ -0,0 +1,55 @@
+import {altair} from "@chainsafe/lodestar-types";
+import PeerId from "peer-id";
+import {ClientKind} from "./peers/client";
+import {Encoding} from "./reqresp/types";
+
+type PeerIdStr = string;
+
+export enum RelevantPeerStatus {
+  Unknown = "unknown",
+  relevant = "relevant",
+  irrelevant = "irrelevant",
+}
+
+export type PeerData = {
+  lastReceivedMsgUnixTsMs: number;
+  lastStatusUnixTsMs: number;
+  connectedUnixTsMs: number;
+  relevantStatus: RelevantPeerStatus;
+  direction: "inbound" | "outbound";
+  peerId: PeerId;
+  metadata: altair.Metadata | null;
+  agentVersion: string | null;
+  agentClient: ClientKind | null;
+  encodingPreference: Encoding | null;
+};
+
+/**
+ * Make data available to multiple components in the network stack.
+ * Due to class dependencies some modules have circular dependencies, like PeerManager - ReqResp.
+ * This third party class allows data to be available to both.
+ *
+ * The pruning and bounding of this class is handled by the PeerManager
+ */
+export class NetworkGlobals {
```

Comment:
Maybe this would be better called `PeerMetadata` and `peerMetadata` throughout.

Thoughts?

Reply:
we have consensus MetaData already, so maybe make this one `PeersData`? there is internal `PeerData` inside

Reply:
would still like to see the name changed to be more specific than "network globals"